opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -297,6 +297,24 @@ enum AggregationTemporality {
   AGGREGATION_TEMPORALITY_CUMULATIVE = 2;
 }
 
+// DataPointFlags is defined as a protobuf 'uint32' type and is to be used as a
+// bit-field representing 32 distinct boolean flags.  Each flag defined in this
```

Comment:
Nit: it should not necessarily be boolean flags only in the future. If needed some flags may use more than one bit. It is probably obvious so I think what is written is fine.